GOVERNANCE.md

```diff
@@ -0,0 +1,105 @@
+
+### Supermajority
+
+A supermajority is defined as two-thirds of members in the group.
+A supermajority of [Maintainers](#maintainers) is required for certain
+decisions as outlined above. Voting on decisions can happen on the mailing list, GitHub, Slack, email, or via a voting service, when appropriate. Maintainers can either vote "agree, yes, +1", "disagree, no, -1", or "abstain". A vote passes when supermajority is met. An abstain vote equals not voting at all.
+
+### Decision Making
+
+Ideally, all project decisions are resolved by consensus. If impossible, any
+maintainer may call a vote. Unless otherwise specified in this document, any
+vote will be decided by a supermajority of maintainers.
+
+Votes by maintainers belonging to the same company
+will count as one vote; e.g., 4 maintainers employed by fictional company **ESOtum** will
+only have **one** combined vote. If voting members from a given company do not
+agree, the company's vote is determined by a supermajority of voters from that
+company. If no supermajority is achieved, the company is considered to have
+abstained.
+
+## Proposal Process
+
+One of the most important aspects in any open source community is the concept
+of proposals. Large changes to the codebase and / or new features should be
+preceded by a proposal in our community repo. This process allows for all
+members of the community to weigh in on the concept (including the technical
+details), share their comments and ideas, and offer to help. It also ensures
+that members are not duplicating work or inadvertently stepping on toes by
+making large conflicting changes.
+
+The project roadmap is defined by accepted proposals.
+
+Proposals should cover the high-level objectives, use cases, and technical
+recommendations on how to implement. In general, the community member(s)
+interested in implementing the proposal should be either deeply engaged in the
+proposal process or be an author of the proposal.
+
+The proposal should be documented as a separated markdown file pushed to the root of the 
+`proposals` folder in the [proposals](https://github.com/external-secrets/external-secrets/proposals)
+repository via PR. The name of the file should follow the name pattern `<short
+meaningful words joined by '-'>_proposal.md`, e.g:
+`clear-old-tags-with-policies_proposal.md`.
+
+Use the [Proposal Template](proposals/TEMPLATE.md) as a starting point.
+
+### Proposal Lifecycle
+
+The proposal PR can be marked with different status labels to represent the
+status of the proposal:
+
+* **New**: Proposal is just created.
+* **Reviewing**: Proposal is under review and discussion.
+* **Accepted**: Proposal is reviewed and accepted (either by consensus or vote).
+* **Rejected**: Proposal is reviewed and rejected (either by consensus or vote).
```

Comment:
I think this part need some tweaking to include the already design/proposal process that we already have in https://github.com/external-secrets/external-secrets/tree/main/design

But we can combine what is being described here and what we already do

Reply:
Will try to fix that tomorrow ! Thanks